WordPress/gutenberg
# Guide: use `code` instead of `keyCode` for keyboard events

## What?


- Refactor the `Guide` component to rely on `code` instead of `keyCode` for keyboard events
- add `preventDefault` to the event callback, so that pressing arrows doesn't scroll the page
- Add unit test for keyboard interactions
- Move page control unit tests to the same file, rewrite them in order to test for user interactions (and not implementation details)

## Why?


[`keyCode`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) is deprecated, and replaced by [`code`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code)

## Testing Instructions





On Storybook, make sure that the `Guide` component:
 - navigates through pages 
 - narrow the page until a horizontal scrollbar appears, and make sure that navigating through the pages with arrow keys doesn't scroll the page's contents
